content/html/concepts/elements/terms/applet/applet.md

```diff
@@ -37,7 +48,39 @@ The `<applet>` element included zero or more [`<param>`](https://www.codecademy.
 </applet>
 ```
 
-## Example
+## Modern Alternatives
```

Comment:
Remove this and the code block above it. It is being repeated.

Reply:
Done ✅